tests/drivers/i2s/i2s_api/src/common.c

```diff
@@ -13,30 +13,30 @@ K_MEM_SLAB_DEFINE(rx_mem_slab, BLOCK_SIZE, NUM_RX_BLOCKS, 32);
 K_MEM_SLAB_DEFINE(tx_mem_slab, BLOCK_SIZE, NUM_TX_BLOCKS, 32);
 
 /* The data_l represent a sine wave */
-ZTEST_DMEM int16_t data_l[SAMPLE_NO] = {
+ZTEST_DMEM int32_t data_l[SAMPLE_NO] = {
```

Comment:
There is no need to change the type of these array's items. They are anyway copied to provided buffers.

Reply:
Yes, see `fill_buf()`.